python/BioSimSpace/Process/_atom_utils.py

```diff
@@ -0,0 +1,977 @@
+######################################################################
+# BioSimSpace: Making biomolecular simulation a breeze!
+#
+# Copyright: 2017-2023
+#
+# Authors: Lester Hedges <lester.hedges@gmail.com>
+#
+# BioSimSpace is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# BioSimSpace is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with BioSimSpace. If not, see <http://www.gnu.org/licenses/>.
+#####################################################################
+__all__ = ["_AToMUtils"]
+from .. import Protocol as _Protocol
+from ..Types import Vector as _Vector
+import math as _math
+import warnings as _warnings
+
+
+class _AToMUtils:
+    # Internal class for creating openmm forces within an AToM process.
+    def __init__(self, protocol):
+        # Check for proper typing
+        if not isinstance(
+            protocol,
+            (
+                _Protocol.AToMMinimisation,
+                _Protocol.AToMEquilibration,
+                _Protocol.AToMAnnealing,
+                _Protocol.AToMProduction,
+            ),
+        ):
```

Comment:
This could also use:

```python
from ..Protocol._atm import _AToM

if not isinstance(_AToM):
    ...
```

Reply:
Now updated (https://github.com/OpenBioSim/biosimspace/blob/feature_AToM/python/BioSimSpace/Process/_atm_utils.py#L33)